static/js/settings_bots.js

```diff
@@ -181,14 +181,16 @@ export function update_bot_settings_tip() {
     const permission_type = bot_creation_policy_values;
     const current_permission = page_params.realm_bot_creation_policy;
     let tip_text;
-    if (current_permission === permission_type.admins_only.code) {
+    if (page_params.is_admin) {
+        $(".bot-settings-tip").hide();
+    } else if (current_permission === permission_type.admins_only.code) {
         tip_text = $t({
             defaultMessage: "Only organization administrators can add bots to this organization.",
         });
     } else if (current_permission === permission_type.restricted.code) {
         tip_text = $t({defaultMessage: "Only organization administrators can add generic bots."});
     } else {
-        tip_text = $t({defaultMessage: "Anyone in this organization can add bots."});
+        $(".bot-settings-tip").hide();
```

Comment:
The code structure can be changed to improve readability. I think we can just return early if we do not want to show the banner after calling `hide`.